corehq/apps/commtrack/tests/util.py

```diff
@@ -164,6 +164,7 @@ def setUp(self):
 
         self.loc = make_loc('loc1')
         self.sp = make_supply_point(self.domain.name, self.loc)
+        self.loc.save()
```

Comment:
Cool, makes sense.

Does this mean that the location should be saved in each place `make_supply_point ` is called? Or may be signal hooks should be modified to take care of this, perhaps `sync_location_supply_point` should be triggered after `make_supply_point` as well?  I am not super familiar with this, @esorgel might have ideas.